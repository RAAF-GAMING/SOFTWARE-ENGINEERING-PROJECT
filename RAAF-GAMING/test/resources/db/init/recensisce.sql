CREATE TABLE IF NOT EXISTS recensisce (
  `cliente` varchar(30) NOT NULL,
  `prodotto` int NOT NULL,
  `voto` smallint NOT NULL,
  `commento` varchar(45) NOT NULL,
  PRIMARY KEY (`cliente`,`prodotto`)
);