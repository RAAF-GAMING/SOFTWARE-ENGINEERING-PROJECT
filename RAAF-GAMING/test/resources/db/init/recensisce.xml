<?xml version="1.0" encoding="UTF-8"?>
<dataset>
    <recensisce cliente='a.delucia19@gmail.com' prodotto="1"  voto="10"  commento="bellissmo"/>
	<recensisce cliente='a.maddaloni10@gmail.com' prodotto="2"  voto="7"  commento="vero veramente"/>
	<recensisce cliente='r.iuliano13@gmail.com' prodotto="1"  voto="10"  commento="awesome"/>
</dataset>