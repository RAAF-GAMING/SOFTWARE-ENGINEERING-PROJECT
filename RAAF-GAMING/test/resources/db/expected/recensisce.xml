<?xml version="1.0" encoding="UTF-8"?>
<dataset>
    <recensisce cliente='a.delucia19@gmail.com' prodotto="1"  voto="10"  commento="bellissmo"/>
	<recensisce cliente='a.maddaloni10@gmail.com' prodotto="2"  voto="7"  commento="vero veramente"/>
	<recensisce cliente='r.iuliano13@gmail.com' prodotto="1"  voto="10"  commento="awesome"/>
	<recensisce cliente='f.peluso25@gmail.com' prodotto="3"  voto="8"  commento="mi pace"/>
</dataset>